elytron/src/test/resources/org/wildfly/extension/elytron/credential-security-factories.xml

```diff
@@ -16,6 +16,7 @@
                                    server="false"
                                    obtain-kerberos-ticket="true"
                                    debug="true"
+                                   mechanisms="KRB5 KRB5LEGACY"
```

Comment:
Sure - just `mechanisms` are primary aimed to well known mechanisms and `mechanism-oids` allows to pass unusual/custom GSS mechanisms.